orion/visualization.py

```
@@ -191,21 +233,92 @@ def _sort_key(m):
             col=1,
         )
 
-        # Mean line
+        # Statistical reference lines and shaded bands
         mean_val = values.mean()
+        std_val = values.std()
+        p99_val = values.quantile(0.99)
+
+        # ±2σ shaded band (wider, subtler shade)
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF003)